extern/storage-sealing/commit_batch.go

```diff
@@ -206,7 +206,25 @@ func (b *CommitBatcher) maybeStartBatch(notif bool) ([]sealiface.CommitBatchRes,
 
 	var res []sealiface.CommitBatchRes
 
-	if total < cfg.MinCommitBatch || total < miner5.MinAggregatedSectors {
+	individual := (total < cfg.MinCommitBatch) || (total < miner5.MinAggregatedSectors)
+
+	if !individual && !cfg.AggregateAboveBaseFee.Equals(big.Zero()) {
```

Comment:
nit: Is the short circuit in case of `AggregateAboveBaseFee` just for efficiency?  It looks like it could be dropped for clarity.

Reply:
Yep, just saving 2 api calls